galasa-managers-parent/galasa-managers-zos-parent/dev.galasa.zos.manager/src/main/java/dev/galasa/zosfile/IZosDataset.java

```diff
@@ -188,7 +188,15 @@ public String toString() {
      * @return data set content
      * @throws ZosDatasetException
      */
-    public String retrieve() throws ZosDatasetException;
+    public String retrieveAsString() throws ZosDatasetException;
```

Comment:
I think I would prefer retrieveAsText() and storeAsText(). Matches with FTP TEXT/BIN options